addons/sale_pdf_quote_builder/models/sale_order.py

```diff
@@ -32,11 +32,11 @@ class SaleOrder(models.Model):
     @api.depends('sale_order_template_id')
     def _compute_available_quotation_document_ids(self):
         for order in self:
-            order.available_quotation_document_ids = self.env['quotation.document'].search(
-                self.env['quotation.document']._check_company_domain(self.company_id),
+            order.available_product_document_ids = self.env['quotation.document'].search(
```

Comment:
```suggestion
            order.available_quotation_document_ids = self.env['quotation.document'].search(
```
I don't think you want to change the field name here :eyes: